sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -565,8 +564,7 @@
     },
     "/jobs": {
       "get": {
-        "summary": "Gets the status of all import/export jobs in an iot hub",
-        "description": "Gets the status of all import/export jobs in an iot hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Gets the status of all import or export jobs in an IoT Hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information.",
```

Comment:
"of all import **and** export jobs.."

Reply:
Good catch. Will change in other instances as well

Reply:
actually not all instances. Sometimes it makes sense to have an 'or'. For example, the 'create' of a job case.